testcafe/tests/drive/classification_scenario.js

```diff
@@ -0,0 +1,395 @@
+    console.group(`\n↳ ℹ️  Loggin & Initialization`)
+    await t.useRole(driveUser)
+    await privateDrivePage.waitForLoading()
+    console.groupEnd()
+  })
+  .after(async ctx => {
+    await ctx.vr.checkRunStatus()
+  })
+
+test(`${TEST_RENAME_FILE}`, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_RENAME_FILE}`)
+  const [fileName, ext] = FILE_PDF.split('.')
+  //set mask only once, all screen in this test use the same mask
+  await t.fixtureCtx.vr.setMaksCoordonnates(maskDriveFolderWithDate)
+
+  await privateDrivePage.goToFolder(`${FEATURE_PREFIX}-Folder2`)
+  await privateDrivePage.renameElementByName(
+    FILE_PDF,
+    `${fileName}2.${ext}`,
+    `${FEATURE_PREFIX}/${TEST_RENAME_FILE}-1`,
+    { exitWithEnter: true }
+  )
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_RENAME_FILE}-1-rename2`,
+    true
+  )
+
+  await privateDrivePage.renameElementByName(
+    `${fileName}2.${ext}`,
+    FILE_PDF,
+    `${FEATURE_PREFIX}/${TEST_RENAME_FILE}-2`,
+    { exitWithEnter: false }
+  )
+
+  await t.fixtureCtx.vr.setMaksCoordonnates(maskDriveFolderWithDate)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_RENAME_FILE}-2-rename2`,
+    true
+  )
+  console.groupEnd()
+})
+
+test(`${TEST_MOVE_FILE_CANCEL}`, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_MOVE_FILE_CANCEL}`)
+  //set mask only once, all screen in this test use the same mask
+  await privateDrivePage.goToFolder(`${FEATURE_PREFIX}-Folder2`)
+
+  console.log('Show Action Menu & Cancel')
+  await privateDrivePage.clickOnActionMenuforElementName(FILE_PDF)
+  await t.fixtureCtx.vr.setMaksCoordonnates(maskDriveFolderWithDate)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_MOVE_FILE_CANCEL}-1`,
+    true
+  )
+  await t.pressKey('esc')
+
+  console.log('Show Move Modale, and Cancel (X button)')
+  await privateDrivePage.showMoveModalForElement(FILE_PDF)
+  // move modale show up : we need a specific mask for it
+  await t.fixtureCtx.vr.setMaksCoordonnates(maskMoveModale)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_MOVE_FILE_CANCEL}-2`,
+    true
+  )
+  await isExistingAndVisibile(moveModale.modalClose, 'Modal button close')
+  await t.click(moveModale.modalClose)
+  await t.fixtureCtx.vr.setMaksCoordonnates(maskDriveFolderWithDate)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_MOVE_FILE_CANCEL}-3`,
+    true
+  )
+  //Remove comment below when https://trello.com/c/RoNPTPZV/1692-modale-d%C3%A9placement-touch-echap-ne-ferme-pas-la-modale is fixed
+  // console.log('Show Move Modale, and Cancel (esc)')
+  // await privateDrivePage.showMoveModalForElement(FILE_PDF)
+  // // no need to screenshot again the modal
+  // await t.pressKey('esc')
+  // await t.fixtureCtx.vr.takeScreenshotAndUpload(
+  //   `${FEATURE_PREFIX}/${TEST_MOVE_FILE_CANCEL}-4`,
+  //   true
+  // )
```

Comment:
Why is this section commented out?

Reply:
Because the bug https://trello.com/c/RoNPTPZV/1692-modale-d%C3%A9placement-touch-echap-ne-ferme-pas-la-modale is not fixed, so if I uncomment it, the test will failed
(I wrote to remove this comment in the card)